app/models/person_demographics_group.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Class representing a group of demographic information for a person.
+# This class is embedded in any model that declares itself as 'demographable'.
+class PersonDemographicsGroup
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  # @!attribute [rw] demographable
+  #   @return [Object] The object that this PersonDemographicsGroup instance is associated with.
+  #   This is a polymorphic association, so it can be any object that declares itself as 'demographable'.
+  embedded_in :demographable, polymorphic: true
+
+  # @!attribute [rw] ethnicities
+  #   @return [Array<Ethnicity>] The ethnicities associated with this PersonDemographicsGroup.
+  embeds_many :ethnicities, class_name: 'Ethnicity', cascade_callbacks: true, validate: true
+
+  # @!attribute [rw] races
+  #   @return [Array<Race>] The races associated with this PersonDemographicsGroup.
+  embeds_many :races, class_name: 'Race', cascade_callbacks: true, validate: true
```

Comment:
Same comment as above for ethnicities field.  Without context someone may be confused if they should pull/persist R/E data using these models.